src/main/resources/chess.css

```diff
@@ -0,0 +1,71 @@
+Label {
+    -fx-border: 1;
+    -fx-min-width: 75;
+    -fx-text-alignment: center;
+    -fx-min-height: 75;
+    -fx-border-color:black;
+    -fx-border-style: solid;
+    -fx-padding: 8 8 8 8;
+    -fx-background-color: #FFFFFF;
+    -fx-alignment: center;
+}
+
+Label.light {
+    -fx-background-color: #ffce9e;
+}
+
+Label.dark {
+    -fx-background-color: #d18b47;
+}
+
+#light0, #dark7 {
+    -fx-background-image: url('images/b_rook.png');
```

Comment:
How should we "move" pieces?

Reply:
Implemented that in a later commit - it would be done by removing the background image (piece) from the origin square and adding a background image to the destination square when a legal move is done.